tests/components/conftest.py

```diff
@@ -9,13 +11,12 @@
 from homeassistant.const import STATE_OFF, STATE_ON
 from homeassistant.core import HomeAssistant
 
-from tests.components.conversation import MockAgent
-
 if TYPE_CHECKING:
-    from tests.components.device_tracker.common import MockScanner
-    from tests.components.light.common import MockLight
-    from tests.components.sensor.common import MockSensor
-    from tests.components.switch.common import MockSwitch
+    from .conversation import MockAgent
```

Comment:
I moved `MockAgent` into `TYPE_CHECKING` as on a new container this caused failures with `hassil` not being installed.